app/views/admin/event_teammate/index.html.haml

```diff
@@ -21,7 +21,7 @@
         = text_field_tag :email, '', class: 'form-control', placeholder: "EventTeammate's email"
       .form-group
         = f.label :role
-        = f.select :role, ['reviewer', 'organizer'], class: 'form-control'
+        = f.select :role, ['reviewer', 'program team', 'organizer'], class: 'form-control'
```

Comment:
These role strings are repeated in a few places in the app, and should probably be made into a constant on some model (probably on User).

```
class User < ActiveRecord::Base
  STAFF_ROLES = ['reviewer', 'program team', 'organizer']
  . . .
end
```

```
= f.select :role, User.STAFF_ROLES, class: 'form-control'
```